packages/core/types/index.js

```diff
@@ -913,7 +913,11 @@ export interface MutableBundleGraph extends BundleGraph<Bundle> {
   getParentBundlesOfBundleGroup(BundleGroup): Array<Bundle>;
   getTotalSize(Asset): number;
   /** Remove all "contains" edges from the bundle to the nodes in the asset's subgraph. */
-  removeAssetGraphFromBundle(Asset, Bundle): void;
+  removeAssetGraphFromBundle(
+    Asset,
+    Bundle,
+    ?$Shape<{|checkReachability: boolean|}>,
```

Comment:
```suggestion
    ?{checkReachability: boolean, ...}>,
```
I think that is equivalent?

Reply:
An empty object is incompatible with that unfortunately: https://flow.org/try/#0PQKgBAAgZgNg9gdzCYAoVAKA3gXwFxhYA+AHgPwEB2ArgLYBGApgE5E4CUQA

Reply:
🙈  Unsealed objects should just be removed